lib/ingram_micro/elements/sales_order_line_item_name_value.rb

```diff
@@ -0,0 +1,33 @@
+module IngramMicro
+  class SalesOrderLineItemNameValue < BaseElement
+    ATTRIBUTE_NAMES = [
```

Comment:
They're the acceptable values for a different element, which perhaps should be implied more in the constant name.